src/context/WishListContext.js

```diff
@@ -0,0 +1,37 @@
+import { createContext, useContext,  useEffect,  useState } from "react";
+
+const WishListContext = createContext(null);
+
+const WishListProvider = ({children}) => {
+    
+    const [ wishlist, setWishList ] = useState(JSON.parse(localStorage.getItem ("wishlist")));
```

Comment:
Suggestion: Can try not storing in local storage.

Reply:
i will try later